src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
 // write code here
+
+const items = document.querySelector('li');
+const tree = document.querySelector('.tree');
+
+for (const item of items) {
```

Comment:
The `for...of` loop is attempting to iterate over `items`, which is not iterable because `items` is a single element. Ensure `items` is a NodeList by using `document.querySelectorAll`.